src/test/java/redis/clients/jedis/resps/RespsTest.java

```diff
@@ -0,0 +1,86 @@
+package redis.clients.jedis.resps;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.*;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import redis.clients.jedis.GeoCoordinate;
+
+public class RespsTest {
+
+    @Test
+    public void KeyedListElementTest() {
+        // test equals
+        KeyedListElement elem = new KeyedListElement("key-name", "elem");
+        KeyedListElement elem_copy = new KeyedListElement("key-name", "elem");
+        assertEquals(elem, elem);
+        assertEquals(elem, elem_copy);
+        assertNotEquals(elem, new Object());
+
+        // test toString
+        String toStringResult = elem.toString();
+        assertThat(toStringResult, CoreMatchers.containsString("key-name"));
+        assertThat(toStringResult, CoreMatchers.containsString("elem"));
+
+        // test hashCode
+        assertEquals(elem.hashCode(), elem_copy.hashCode());
+    }
+
+    @Test
+    public void KeyedZSetElementTest() {
+        // test equals
+        KeyedZSetElement elem = new KeyedZSetElement("key-name", "elem", 1d);
+        KeyedZSetElement elem_copy = new KeyedZSetElement("key-name", "elem", 1d);
+        assertEquals(elem, elem);
+        assertEquals(elem, elem_copy);
+        assertNotEquals(elem, new Object());
+
+        // test toString
+        String toStringResult = elem.toString();
+        assertThat(toStringResult, CoreMatchers.containsString("key-name"));
+        assertThat(toStringResult, CoreMatchers.containsString("elem"));
+        assertThat(toStringResult, CoreMatchers.containsString("1"));
+
+        // test hashCode
+        assertEquals(elem.hashCode(), elem_copy.hashCode());
+    }
+
+    @Test
+    public void TupleTest() {
```

Comment:
There should be an existing TupleTest class. If that class covers this method, delete this. If not, move this there.